modules/client/examples/02-tokenVoting-client/07-get-proposal.ts

```diff
@@ -1,30 +1,32 @@
 /* MARKDOWN
-### Retrieve a proposal by proposalID (TokenVoting)
+### Get a proposal by proposalID (TokenVoting)
 
-Retrieving the proposals of a TokenVoting DAO.
+Gets a specific proposal using the TokenVoting plugin as its governance mechanism.
 */
+
 import {
-  Context,
   ContextPlugin,
   TokenVotingClient,
-  TokenVotingProposal,
+  TokenVotingProposal
 } from "@aragon/sdk-client";
-import { contextParams } from "../00-client/00-context";
+import { context } from "../00-setup/00-getting-started";
 
-// Create a simple context
-const context: Context = new Context(contextParams);
-// Create a plugin context from the simple context
+// Instantiate a plugin context from an Aragon OSx SDK context.
 const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
-// Create a TokenVoting client
-const client = new TokenVotingClient(contextPlugin);
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(contextPlugin);
+
+// The address of the proposal you want to retrieve.
+const proposalId: string = "0x12345238513498562394651375914323423";
```

Comment:
El formato correcto de proposal id seria algo del estilo `0x12345238513498562394651375914323423_0x1`

Reply:
Porque? jaj que es el `_0x1`?

Reply:
Is this for every time a `proposalId` is called?

Reply:
The proposal ID includes the plugin address and the proposal counter, so that the plugin knows which one it is. There is no `the address of the proposal`, it's simply the proposal ID

Reply:
Will fix for all then! Thanks for the insight @brickpop